core/src/builder/transaction/mod.rs

```diff
@@ -741,23 +741,63 @@ pub fn create_replacement_deposit_txhandler(
         .add_output(UnspentTxOut::from_partial(anchor_output())))
 }
 
-/// Helper function to create a taproot output that combines a script and a root hash
+pub fn create_disprove_taproot_output(
```

Comment:
can we have some docs explaining the parameters? I'm assuming hidden_node is the rest of the disprove scripts, first arg seems to be operator's timeout spend script, additional is additional_disprove_scripts then?